app/presenters/coronavirus_landing_page_presenter.rb

```diff
@@ -28,6 +27,47 @@ def faq_schema(content_item)
     }
   end
 
+  def topic_section
+    { links: [
+      {
+        url: "/search/all?level_one_taxon=5b7b9532-a775-4bd2-a3aa-6ce380184b6c&content_purpose_supergroup%5B%5D=guidance_and_regulation&order=updated-newest",
+        label: "Guidance and regulation about COVID-19",
```

Comment:
This is a real nitpick, and not a blocker, but I wonder why you chose to structure the hash as "url, label" rather than "label, url" as it is in the other methods.
e.g.
```ruby
{
    label: "Guidance and regulation about COVID-19",
    url: "/search/all?level_one_taxon=5b7b9532-a775-4bd2-a3aa-6ce380184b6c&content_purpose_supergroup%5B%5D=guidance_and_regulation&order=updated-newest",
}
```

The YAML is also "label, url" so this made checking the content such fun! 😬

Reply:
🤣 no worries - fixed up